src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentDay = 40;
+  const total = rentDay * days;
+
+  if (days >= 7) {
+    return total - 50;
```

Comment:
Use a constant variable for the discount amount (e.g. `const bigDiscount = 50;`), so it's easier to modify and understand the code in the future 🚀